src/nvim/testdir/unix.vim

```diff
@@ -10,6 +10,3 @@ endif
 
 " Don't depend on system locale, always use utf-8
 set encoding=utf-8
-
-" Use safer defaults for various directories
-set backupdir=. directory=. undodir=. viewdir=.
```

Comment:
User files could be overwritten

Reply:
Yes.. it was more meant as a test to see how this behaves when removed etc.

In general I think `set noswapfile` makes more sense than `directory=.`.
Otherwise it should follow a safe setting of `$HOME`, shouldn't/doesn't it?

Reply:
What do you suggest since we cannot have `set directory=.` as is?
Leave only that out for now?
(Related: https://github.com/vim/vim/pull/4742)

Reply:
can it be set a a temp dir?